packages/snap-preact/src/Snap.tsx

```diff
@@ -211,7 +211,50 @@ export class Snap {
 		return this.controllers[config.id];
 	};
 
+	public beaconErrorEvent = (event: ErrorEvent): void => {
```

Comment:
Can we use `handlers` similar to what was don in AC Controller?

Eg: `handlers.error`